src/app/js/formats/streamgraph/utils.js

```diff
@@ -0,0 +1,204 @@
+
+import differenceBy from 'lodash/differenceBy';
+import * as d3 from 'd3';
+
+// ===========================================================================================
+// getCssToString get the css js object and return it as string
```

Comment:
Please, remove this kind of comment.
They take too much space and if the function is small enough and well named, we don't need them.